### ISSUE TITLE
Braille sense U2 QWERTY bugs 

### ISSUE THREAD

**Issue by Albarra45** [nvaccess/nvda#6006]
Braille Sense U2 QWERTY notetaker does not work with quick launch keys in file manager after I connected to NVDA.
For instance, when I press the Windows key on the QWERTY input keys, I can not use shortcut keys to go word Processor by pressing Windows key+w, scientific calculator, etc. 
This may be a bug.
also, when I press left or right display keys, NVDA stops speaking. 
I want to read it on the braille display with NVDA speaks on the screen at the same time.

If you have any questions,, please let me know.

Thanks,
Albarra

**Comment by leonardder** [comment]
Hi,

I might have a BS QWERTY available in the near future to test this. Could you give a more detailed explanation of the steps leading to your issue?

**Comment by Albarra45** [comment]
Hello, 

 

After the Bs QWERTY connect to NVDA by USB cable, I would like to press Window key to go to File Manager. If you press Cap Lock + s to go to Terminal Mode, it dos not work. That is an example of the Quick Lauch command issue. In order to fix it, you disconnect NVDA and press reset key on your BS QWERTY. 

It is unusual for me. I have no problem with other screen readers. 

There is an another example for Quick Launch command keys while the BS QWERTY connect to NVDA. Press Windows key to go to File Manager and press Windows-key+w to go to Word Processor. It doesn’t work as well. This may be a bug.

Please let me know if you can reproduce this issue. 

 

 

Just your information, NVDA does  not support braille QWERTY input keys. In other words, I cannot press capitalized letters on my QWERTY Keyboard. I have to use computer keyboard instead. 

Also, I can not use NVDA command keys for Braille QWERTY as well. I am very pleased that NVDA developers can support and improve the Braille QWERTY input keys.

 

 

Thank you,

Albarra

**Comment by ehollig** [comment]
@leonardder, do you have a BS QWERTY available to test this? https://github.com/nvaccess/nvda/issues/6006#issuecomment-226952950 says you may.

**Comment by Albarra45** [comment]
Yes I have it.

**Comment by leonardder** [comment]
I'm able to reproduce it. I assume it has to do with the keyboard keys being captured to send input to the screen reader, and the capture isn't released when leaving terminal mode.

I'm afraid the only way to fix this is abandoning the possibility to use the qwerty keyboard to control NVDA. This would be a work around, no real fix for this problem. This issue probably needs to be reported to Hims, and since the Polaris has taken al their priorities, it is very unlikely to be fixed.

**Comment by leonardder** [comment]
Since Hims is only fixing very critical bugs in their Sense firmware (i.e. they aren't planning on new releases with new functionality), I'm closing this as a can't fix.
